src/audio/module_adapter/module/generic.c

```diff
@@ -191,7 +191,7 @@ void *mod_alloc_align(struct processing_module *mod, uint32_t size, uint32_t ali
 	if (alignment)
 		ptr = rballoc_align(SOF_MEM_FLAG_USER, size, alignment);
 	else
-		ptr = rballoc(SOF_MEM_FLAG_USER, size);
+		ptr = rmalloc(SOF_MEM_FLAG_USER, size);
```

Comment:
Should we also ```rmalloc_aligned()``` above for ```rballoc_aligned()```

Reply:
I created rmalloc_align() and took that into use.